test/doc/algo/reduce.cpp

```diff
@@ -1,5 +1,6 @@
 #include <eve/module/core.hpp>
 #include <eve/algo.hpp>
+#include <numeric>
 #include <iostream>
 #include <vector>
 #include "print.hpp"
```

Comment:
due to how doc tests are able to be opened in Godbolt, we should not use print.hpp in those doc tests.
What I think is OK is to reproduce of the std cppreference samples, rise the size of the data and be happy.

Again, check how copy.cpp has been modified